idris-commands.el

```diff
@@ -238,7 +238,10 @@ line."
            (lambda (_condition)
              (when (member 'warnings-tree idris-warnings-printing)
                (idris-list-compiler-notes)
-               (pop-to-buffer (idris-buffer-name :notes)))))))
+	       (if idris-stay-in-current-window-on-compiler-error
+                 (display-buffer (idris-buffer-name :notes))
+                 (pop-to-buffer (idris-buffer-name :notes)))
+	       )))))
```

Comment:
no need to create a new PR, do your commits and then a git rebase -i origin/master, mark all but the first one as fixup.  and then force-push